x-pack/plugins/apm/server/lib/helpers/create_es_client/create_apm_event_client/unpack_processor_events.ts

```diff
@@ -21,13 +20,24 @@ const processorEventIndexMap = {
   [ProcessorEvent.profile]: 'transaction',
 } as const;
 
+export function processorEventsToIndex(
+  events: ProcessorEvent[],
+  indices: ApmIndicesConfig
+) {
+  return uniq(events.map((event) => indices[processorEventIndexMap[event]]));
```

Comment:
Will this get rid of duplicate `apm-*,apm-*,apm-*`? I thought this was fixed already. If not, thanks!

Reply:
on line 39 we `uniq` the events themselves (which is useful for cases where we might pass in the transaction or metric indices), but the index names themselves were not deduped, so that should be fixed now as well.

Reply:
(the events themselves are used for creating the terms query so those need to be deduped separately)